src/engine-actions/get-status.spec.js

```diff
@@ -18,12 +18,14 @@ describe('get-status', () => {
     beforeEach(() => {
       getInfoResponse = {
         chains: [{ chain: 'bitcoin', network: 'testnet' }],
+        version: '0.7.1',
```

Comment:
is this the format that LND returns? I would think it would have `-beta` in it

Reply:
Yeah, it currently returns beta but the comparison function strips that part off. The stripping is tested in other unit tests.